django_mongodb_backend/management/commands/showencryptedfieldsmap.py

```diff
@@ -0,0 +1,37 @@
+from bson import json_util
+from django.apps import apps
+from django.core.management.base import BaseCommand
+from django.db import DEFAULT_DB_ALIAS, connections, router
+
+from django_mongodb_backend.utils import model_has_encrypted_fields
+
+
+class Command(BaseCommand):
+    help = """
+    Shows the mapping of encrypted fields to field attributes, including data
+    type, data keys and query types. The output can be used to set
+    ``encrypted_fields_map`` in ``AutoEncryptionOpts``.
+
+    Defaults to showing keys from the ``key_vault_namespace`` collection.
```

Comment:
Is this sentence leftover from --create-data-keys? (if not, what is the non-default behavior?)